bootcamp/tutorials/integration/llamaindex/llamaindex_milvus_full_text_search.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>simply&nbsp;trying<a href="https://docs.zilliz.com/docs/register-with-zilliz-cloud" target="_blank"> Zilliz Cloud</a> for free</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1516/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>